lib/lang/latin/data/verb/suffixes.csv

```diff
@@ -706,4 +706,4 @@ tus esse,3rd,passive,infinitive,perfect,,,,regular,
 ātum īrī,1st,passive,infinitive,future,,,,regular,
 itum īrī,2nd,passive,infinitive,future,,,,regular,
 tum īri,3rd,passive,infinitive,future,,,,regular,
-ītum īrī,4th,passive,infinitive,future,,,,regular,
+ītum īrī,4th,passive,infinitive,future,,,,regular
```

Comment:
actually we need the trailing comma to indicate that we have an empty column. See my comments in #107 which will fix this via #108